### PR TITLE
chore: update paragon and frontend-build to openedx

### PR DESCRIPTION
**Pull request by Yagnesh1998**
There is an edx organization on NPM that currently publishes many packages. An organization can't be renamed but there is a process for moving packages between organizations. The edx organization or scope as NPM calls it, has both packages that are a part of the Open edX ecosystem as well as packages that are specific to edx.org. To reduce confusion we'll be moving the openedx packages to a new openedx scope in NPM.

**Impact**
Any downstream packages that depend on a package that has moved will need to update its dependencies and imports to pull the same files from the new package location and version.

Part of https://github.com/openedx/axim-engineering/issues/23

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @Yagnesh1998! Please note that it may take us up to several weeks or months to complete a review and merge your PR.

Feel free to add as much of the following information to the ticket as you can:
- supporting documentation
- [Open edX discussion forum threads](https://discuss.openedx.org/)
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).


This is currently a draft pull request. When it is ready for our review and all tests are green, click "Ready for Review", or remove "WIP" from the title, as appropriate.

File: src/index.test.jsx [hunk 1]
```diff
@@ -6,7 +6,7 @@ import {
   initialize,
   mergeConfig,
   subscribe,
-} from '@edx/frontend-platform';
+} from '@@/frontend-platform';
```
Comment (brian-smith-tcril) [inline src/index.test.jsx:9]:
```suggestion
} from '@openedx/frontend-platform';
```

**Comment by brian-smith-tcril** [comment]
Thank you so much for doing this @Yagnesh1998!

I tried pulling this locally and running
```
nvm use
npm ci
```

and it failed. I then deleted the `package-lock.json` file and ran `npm install` and was able to run the MFE.

I then ran `npm test` and all but 1 test passed. It seems that was just a find/replace error here https://github.com/openedx/frontend-app-learner-dashboard/pull/285/files#r1481954243

If you:
* apply my linked suggestion
* rebase this PR on latest master
* delete/regenerate the `package-lock.json`

then this should be good to move out of draft!

**Comment by openedx-webhooks** [comment]
@Yagnesh1998 Even though your pull request wasn’t merged, please take a moment  to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=openedx/frontend-app-learner-dashboard&entry.867055334=https://github.com/openedx/frontend-app-learner-dashboard/pull/285&entry.1484655318=https://github.com/Yagnesh1998&entry.752974735=2024-02-07+16:35&entry.1917517419=2024-02-07+20:47&entry.2133058324=No) so we can improve your experience in the future.

**Comment by openedx-webhooks** [comment]
@Yagnesh1998 Even though your pull request wasn’t merged, please take a moment  to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=openedx/frontend-app-learner-dashboard&entry.867055334=https://github.com/openedx/frontend-app-learner-dashboard/pull/285&entry.1484655318=https://github.com/Yagnesh1998&entry.752974735=2024-02-07+16:35&entry.1917517419=2024-02-07+20:47&entry.2133058324=No) so we can improve your experience in the future.

**Pull request closed without merging**